backend/__tests__/cronJobs.test.ts

```diff
@@ -0,0 +1,122 @@
+import { expect } from 'chai';
+import mongoose from 'mongoose';
+import { MongoMemoryServer } from 'mongodb-memory-server';
+import ComplianceReport from '../src/db/schemas/compliance';
+import { startCronJobs } from '../src/cronJobs';
+import * as removeExpiredDrafts from '../src/cronJobs/removeExpiredDrafts';
+import { appConfig } from '../src/config';
+import sinon from 'sinon';
+
+describe('Cron Job', () => {
+    let mongod;
+    let clock;
+    let originalCronSchedule = appConfig.cron.removeExpiredDraftsSchedule;
+
+    before(async () => {
```

Comment:
can you add descritpion that it's a cron job test setup?

Reply:
done